### PR TITLE
Milestone 1: Copy of NewsWeek.com website using Bootstrap.

### PR DESCRIPTION
**Pull request by tiagoit**
# Copy of NewsWeek.com (With Bootstrap)

Copy of NewsWeek.com website using Bootstrap.

### Technology

HTML, CSS, Bootstrap

### Authors

- **Tiago Ferreira** - [@ferreirati](https://github.com/ferreirati)

### Links

- **Assignment page** - [The Odin Project](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap)
- **Live** - [Github pages](https://ferreirati.github.io/mv-07-htmlcss-newsweek/src)

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# Pull  request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
Hello Tiago,👋
Nice first attempt on this project.👌
There are a few corrections to be made to get this project approved😔 
#### **[REQUIRE] NAVBAR🙂
It changes with RWD breakpoints.
> Check carefully how the navbar behaves in the actual [newsweek page](newsweek.com)
how it behaves at extra small, SM, MD, and LG breakpoint. currently, it works well at Lg and MD but not on XS(COL-"VALUE") 🙂

#### **[REQUIRE] MAIN🙂
Check carefully how the contents such as Featured Stories, Top Story, Opinions behave at different media breakpoints. (especially at extra small breakpoint)

> some come ahead of others depending on the breakpoint
> Take note of how each column is ordered. [read](https://getbootstrap.com/docs/4.0/layout/grid/)
> Note the order of bootstrap 4 breakpoint ```lg``` ```md``` ```sm``` and ```col```(formally xs in bootstrap3)

#### **[REQUIRE] implementation
Fix the horizontal scroll on the page
> this arises as a result of the wrong implementation of ```.container``` ```row``` and ```col```

# Kindly make those changes and submit for review again
Happy coding🙂🙂

**Comment by tiagoit** [comment]
> Hello Tiago,
> Nice first attempt on this project.
> There are a few corrections to be made to get this project approved
> 
> #### **[REQUIRE] NAVBAR
> It changes with RWD breakpoints.
> 
> > Check carefully how the navbar behaves in the actual [newsweek page](newsweek.com)
> > how it behaves at extra small, SM, MD, and LG breakpoint. currently, it works well at Lg and MD but not on XS(COL-"VALUE") slightly_smiling_face
> 
> #### **[REQUIRE] MAIN
> Check carefully how the contents such as Featured Stories, Top Story, Opinions behave at different media breakpoints. (especially at extra small breakpoint)
> 
> > some come ahead of others depending on the breakpoint
> > Take note of how each column is ordered. [read](https://getbootstrap.com/docs/4.0/layout/grid/)
> > Note the order of bootstrap 4 breakpoint `lg` `md` `sm` and `col`(formally xs in bootstrap3)
> 
> #### **[REQUIRE] implementation
> Fix the horizontal scroll on the page
> 
> > this arises as a result of the wrong implementation of `.container` `row` and `col`
> 
> # Kindly make those changes and submit for review again
> Happy coding

Thanks for the code review Forison. I've implemented all the fixes.

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+          <span>U.S. Edition</span>
+          <span>日本</span>
+          <span>한국</span>
+          <span>Pakistan</span>
+          <span>Polska</span>
+          <span>România</span>
+        </div>
+      </div>
+    </footer>
+  </main>
```
Comment (Redvanisation) [inline src/index.html:816]: The `<footer>` tag shouldn't be inside the `<main>` tag each one of represents a different part of the website content.

**Review by Redvanisation — APPROVED** [review]
### Hello @ferreirati   🌟

### Very well done on this project, you have met the project requirements so I am giving you a thumb's up 👍  

- Please pay attention to the comment I made below about the **semantic HTML tags** and make sure you take into consideration in your future projects.

### _You can merge to master and submit a project completion form! 🚀_ 

### Good luck! 🍀

File: src/index.html (hunk 1 above)
Comment (tiagoit) [reply src/index.html:816]: Thanks for the feedback Redvanisation, I'll read some more about semantic HTML.

**Pull request merged**